src/relcast.erl

```diff
@@ -454,7 +479,7 @@ handle_actions([{multicast, Message}|Tail], Batch, State =
                #state{key_count=KeyCount, bitfieldsize=BitfieldSize, id=ID, ids=IDs, active_cf=CF, module=Module}) ->
     Bitfield = make_bitfield(BitfieldSize, IDs, ID),
     Key = make_outbound_key(KeyCount),
-    rocksdb:batch_put(Batch, CF, Key, <<0:1/integer, Bitfield:BitfieldSize/bits, Message/binary>>),
+    rocksdb:batch_put(Batch, CF, Key, <<0:2/integer, Bitfield:BitfieldSize/bits, Message/binary>>),
```

Comment:
Wise to put an `ok=rocksdb:...` or case clause this? Although it looks like it can only return an ok from the batch_put spec or cause a nif error.